compiler/rustc_codegen_llvm/src/gotoc/operand.rs

```diff
@@ -547,9 +549,11 @@ impl<'tcx> GotocCtx<'tcx> {
     pub fn codegen_func_expr(&mut self, instance: Instance<'tcx>, span: Option<&Span>) -> Expr {
         let func = self.symbol_name(instance);
         let funct = self.codegen_function_sig(self.fn_sig_of_instance(instance));
+        let pretty_name = self.readable_instance_name(instance).clone();
```

Comment:
why is the clone needed

Reply:
The borrow checker was yelling at me because ensure consumes a mutable reference, but the right thing is to use the ctx passed through the closure.